### DISCUSSION TITLE
Prepared configurations (templates) for some apps

### DISCUSSION THREAD

**Discussion by ceskyDJ** [ImranR98/Obtainium#873] [General]
Hi Obtainium lovers! As you've probably noticed before, there are some apps with more complicated configuration in Obtainium. These don't use GitHub releases, has more versions (e.g. with/without some functionality), etc. I decided to make this discussion, where I create a list of apps I've already configured properly, so anyone could just copy this configuration into own Obtainium instance and use it without additional effort. If you want to add some app to the list, you could send working configuration in your comment.

The list contains only configuration options that **needs to be changed**! Everything else should stay with default values (mostly empty) or values set by app, when entering manual options (mostly App Source URL).

# General Android phone/tablet (arm64)

Tested on: **Samsung Galaxy S22** with **Android 13**

## Collabora Office
![collabora-office](https://github.com/ImranR98/Obtainium/assets/48548230/6cf3f583-8089-489b-a1ad-c798c09a6eb5)

**App Source URL**: `https://collaboraoffice.com/downloads/Collabora-Office-Android-Snapshot/?C=M;O=D`
**Version Extraction RegEx**: `-([0-9]{4}-[0-9]{2}-[0-9]{2})\.apk$`
**Match  Group to Use**: `1`
**Filter APKs by Regular Expression**: `.+-snapshot-arm64-.+`

## DAVx5
![davx5](https://github.com/ImranR98/Obtainium/assets/48548230/f81881b3-3bcb-42b6-88ad-acd2330c7cdf)

**App Source URL**: `https://github.com/bitfireAT/davx5-ose`
**App Name**: `DAVx5` (original name has problems with displaying correctly)

## Saber
![saber](https://github.com/ImranR98/Obtainium/assets/48548230/24e25a8e-3b95-44c1-8b37-7c2ce5707ad2)

**App Source URL**: `https://github.com/adil192/saber`
**Filter APKs by Regular Expression**: `Saber_v.+`

## Megalodon
![megalodon](https://github.com/ImranR98/Obtainium/assets/48548230/fe2fe8c6-e4b8-4592-980c-b71103ee7b59)

**App Source URL**: `https://github.com/sk22/megalodon`
**Filter APKs by Regular Expression**: `megalodon.apk`

## Home Assistant
![home-assistant](https://github.com/ImranR98/Obtainium/assets/48548230/3a589d04-3da9-4bd9-a911-5da2c88e9f60)

**App Source URL**: `https://github.com/home-assistant/android`
**Filter APKs by Regular Expression**: `app-full-release.apk`

## Habits
![habits](https://github.com/ImranR98/Obtainium/assets/48548230/0babe71c-3de6-4902-b6f8-582c1c8b574e)

**App Source URL**: `https://github.com/iSoron/uhabits`
**Filter APKs by Regular Expression**: `-release.apk$`

## Bitwarden
![bitwarden](https://github.com/ImranR98/Obtainium/assets/48548230/7115fff1-42c7-4442-9584-86a8f5680b5c)

**App Source URL**: `https://github.com/bitwarden/mobile`
**Filter APKs by Regular Expression**: `bitwarden\.apk$`

**Comment by ImranR98** [comment]
There's now a public site for this which you might be interested in contributing to. See #1214.
